src/Pokemon.java

```diff
@@ -0,0 +1,47 @@
+public abstract class Pokemon {
+    // Een Super klasse waarvan minimaal 2 private variabelen, 1 constructor, 2 methodes, 2 getters en 2 setters;
```

Comment:
Dit soort comments moet je niet naar je main branch pushen. Als je ze gebruikt tijdens development, is prima, maar verwijder ze naderhand ook. Ook "constructor", "getters" en "setters" zijn overbodige comments. Als je die er in zet voor de structuur, dan is een simpele witregel ook voldoende.